src/condor_utils/file_transfer.cpp

```diff
@@ -4373,6 +4394,29 @@ FileTransfer::computeFileList(
 	ExpandFileTransferList( FilesToSend, filelist, preserveRelativePaths );
 	// dPrintFileTransferList( D_ZKM, filelist, ">>> computeFileList(), after ExpandeFileTransferList():" );
 
+	if (inHandleCommands && m_has_protected_url) {
+		ExprTree * tree = jobAd.Lookup(ATTR_TRANSFER_Q_URL_IN_LIST);
+		if (tree && tree->GetKind() == ClassAd::ExprTree::EXPR_LIST_NODE) {
+			classad::ExprList* list = dynamic_cast<classad::ExprList*>(tree);
+			for(classad::ExprList::iterator it = list->begin() ; it != list->end(); ++it ) {
+				std::string files, attr;
+				classad::Value item;
+				if (jobAd.EvaluateExpr(*it, item, classad::Value::ALL_VALUES) && item.IsStringValue(files)) {
+					if ((*it)->GetKind() == ClassAd::ExprTree::ATTRREF_NODE) {
+						classad::ClassAdUnParser unparser;
+						unparser.SetOldClassAd( true, true );
+						unparser.Unparse(attr, *it);
+						//TODO: De-construct attr for specified xfer queue
+					} else { /*Fail?*/ }
+				} else { /*Fail?*/ }
```

Comment:
The job should probably go on hold.  My thinking is that it's more likely that the job was incorrectly manipulated than that we've discovered a bug or an unchecked version mismatch, but this doesn't need to be fixed right now, I don't think, since it'll only trigger if the magic attribute is set.

Reply:
Move this comment into the related ticket so it is not lost to the ether